icons/checkmk.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Checkmk</title><path d="M5.2,8.7v4l4.9-3.2v8.2L12,18.8l1.9-1.1V9.6l4.9,3.2v-4L12,4.5ZM12,0,22.4,6V18L12,24,1.6,18V6Z"/></svg>
```

Comment:
I downloaded the SVG assets and took `vertical/checkmk_logo_main_vertical.svg` file to do the comparison. Which file did you use exactly?

With this file, I got some differences in the arrow (original in green overlayed by your version in black):
![image](https://user-images.githubusercontent.com/12817388/135491842-98125361-ba1f-456f-b5ec-1d193b00424c.png)

Reply:
Hello @animesh-anand,

Welcome to Github and Simple Icons and thank you for your contribution.

The next step would be top make the requested changes in your branch. Once done, you will need to commit those changes and push them again.

Let me know if there is something in particular you need help with!